packages/browser/src/login/oidc/incomingRedirectHandler/AuthCodeRedirectHandler.ts

```diff
@@ -149,12 +149,16 @@ export class AuthCodeRedirectHandler implements IIncomingRedirectHandler {
       };
     }
 
-    const authFetch = await buildAuthenticatedFetch(fetch, tokens.accessToken, {
-      dpopKey: tokens.dpopKey,
-      refreshOptions,
-      eventEmitter,
-      expiresIn: tokens.expiresIn,
-    });
+    const authFetch = await buildAuthenticatedFetch(
+      window.fetch,
```

Comment:
should this be `window.fetch` or should it be `import { fetch } from "cross-fetch";`? (I mean, yeah, arguably we are in the browser, so we have `window.fetch` but I think maybe we want to standardise around the import, especially with our Node.js 18+ support work.

Reply:
Fixed in 535276759bd27391d0c211ccca72d2531de128f8, but there's a lot of that remaining :)